packages/frontend/src/hooks/usePositions.ts

```diff
@@ -72,23 +72,33 @@ export const usePositions = () => {
           const wethAmt = new BigNumber(isWethToken0 ? s.amount0 : s.amount1)
           const usdAmt = getUsdAmt(wethAmt, s.timestamp)
 
+          const time = new Date(Number(s.timestamp) * 1000).setUTCHours(0, 0, 0) / 1000
+
           //buy one squeeth means -1 to the pool, +1 to the user
           acc.squeethAmount = acc.squeethAmount.plus(squeethAmt.negated())
-          acc.wethAmount = acc.wethAmount.plus(wethAmt.negated())
 
           //<0 means, buying squeeth
           //>0 means selling squeeth
           acc.totalSqueeth = acc.totalSqueeth.plus(squeethAmt)
           acc.totalETHSpent = acc.totalETHSpent.plus(wethAmt)
           acc.totalUSDSpent = acc.totalUSDSpent.plus(usdAmt)
 
-          acc.usdAmount = acc.usdAmount.plus(usdAmt)
+          if (squeethAmt.isPositive()) {
```

Comment:
it seems that the wethAmount from swap data is not equal to the ethCollateral amount
no matter when I open a new short position, or close it partially or close it fully, they are not equal to the eth collateral amount 
cc @alexisgauba

Reply:
Yep see #85